src/main/java/no/nav/familie/integrasjoner/medlemskap/MedlemskapController.kt

```diff
@@ -19,4 +16,8 @@ class MedlemskapController(private val medlemskapService: MedlemskapService) {
         return success(medlemskapService.hentMedlemskapsunntak(aktørId), "Henting av medlemskapsunntak OK")
     }
 
+    @GetMapping("forIdent")
+    fun hentMedlemskapsunntakForIdent(@RequestHeader("Nav-Personident") ident: String): Medlemskapsinfo {
```

Comment:
Ressurs?

Reply:
Helst ikke. Ser fremdeles ikke verdien av det. Mulig den ha noe for seegf i front end, men i maskin til maskin kommunikasjon ser jeg ikke noe mer enn ekstra støy i koden på begge sider.

Reply:
Jeg er ikke størst tilhenger av det selv, men det var det som ble valgt og er tilhenger at man gjør det likt. Slik du gjør det nå vil uansett feilmeldinger bli en Ressurs. 
@henninghaakonsen